src/test/java/ua/com/juja/microservices/acceptance/TeamsAcceptanceTest.java

```diff
@@ -0,0 +1,170 @@
+package ua.com.juja.microservices.acceptance;
+
+import com.lordofthejars.nosqlunit.annotation.UsingDataSet;
+import com.lordofthejars.nosqlunit.core.LoadStrategyEnum;
+import io.restassured.response.Response;
+import net.javacrumbs.jsonunit.core.Option;
+import org.eclipse.jetty.http.HttpMethod;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static ua.com.juja.microservices.teams.service.Utils.convertToString;
```

Comment:
please use static imports only for testing libraries (it's common approach). It's difficult to understand quickly where this method from.